nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/tools/ScanTable.java

```diff
@@ -254,6 +255,8 @@ final protected void selectionChanged() {
             }
             chart.refresh();
 
+        } else if (!Processor.getProcessor().isProcessorAvailable()) {
+            log.warn("Unable to load row. Processor unavailable.");
```

Comment:
not sure if I should also add a popup to the user here as well?

Reply:
I would vote for no popup.
But what happens on selection, do user see the selection works on the tableview? I cannot test yet (asked you on Teams for data to reproduce the bug).
I'm just wondering if preventing user from selecting sth during the processing wouldn't be needed too. I didn't check if it is easily feasible so it is just an idea.

Reply:
Yes on selection the row should be loaded, I think it could also be possible to disable the table from selection while the processor is busy though, I would just need to add a listener